### PR TITLE
Ignore a warning in kubectl apply when namespace is already created.

### LINKED ISSUE #18948: `mesh manifest apply` error
Run the command below to install istio. Before this, I have the namespace already created.

```
istioctl manifest apply
```

I got this error in the end:

```
*** Errors were logged during apply operation. Please check component installation logs above. ***

```

which is caused by:

```
Component Base installed successfully:
======================================
Error detail:

Warning: kubectl apply should be used on resource created by either kubectl create --save-config or kubectl apply (repeated 1 times)





namespace/istio-system configured

```

Related issue in k8s: https://github.com/kubernetes/kubernetes/issues/82154

### PR DESCRIPTION
**Pull request by elfinhe**
Fix: https://github.com/istio/istio/issues/18948

### CODE REVIEW

File: cmd/mesh/manifest.go [hunk 1]
```diff
@@ -146,7 +146,7 @@ func genApplyManifests(setOverlay []string, inFilename string, force bool, dryRu
 			l.logAndPrintf("\n%s\n%s", cs, strings.Repeat("=", len(cs)))
 		}
 
-		if strings.TrimSpace(out[cn].Stderr) != "" {
+		if !isStderrOkay(out[cn].Stderr) {
```
Comment (ostromart) [inline cmd/mesh/manifest.go]: Consider renaming function to ignoreError, which is more meaningful.

File: cmd/mesh/manifest.go [hunk 2]
```diff
@@ -162,6 +162,20 @@ func genApplyManifests(setOverlay []string, inFilename string, force bool, dryRu
 	return nil
 }
 
+func isStderrOkay(stderr string) bool {
+	trimmedStdErr := strings.TrimSpace(stderr)
+	ignoreList := []string{
```
Comment (ostromart) [inline cmd/mesh/manifest.go]: better to put this list in var() block at the top to give it more visibility.

File: cmd/mesh/manifest.go (hunk 1 above)
Comment (elfinhe) [reply cmd/mesh/manifest.go]: Fixed.

File: cmd/mesh/manifest.go (hunk 2 above)
Comment (elfinhe) [reply cmd/mesh/manifest.go]: Fixed.

**Review by ostromart — APPROVED** [review]

**Pull request merged**